.gitattributes

```diff
@@ -0,0 +1 @@
+.packages/** linguist-vendored
```

Comment:
Let's also add this to fastn-template repo, so all future repos automatically get this.

Reply:
@harshdoesdev resolve this comment once this is done.